pyfakefs/fake_filesystem.py

```diff
@@ -1003,7 +1014,7 @@ def SplitDrive(self, path):
                 # UNC path handling is here since Python 2.7.8, back-ported from Python 3
                 if sys.version_info >= (2, 7, 8):
                     if (path[0:2] == self.path_separator * 2) and (
-                            path[2:3] != self.path_separator):
+                                path[2:3] != self.path_separator):
```

Comment:
Didn't notice these changes - PyCharm has sometimes its own ideas about hanging indents, obviously I did an auto-format accidentally. I will revert this.